uk.co.johngabriel.amaya.yml

```diff
@@ -0,0 +1,33 @@
+id: uk.co.johngabriel.amaya
```

Comment:
I need some way to connect https://www.johngabriel.co.uk/ with the git repo for verification. Is the game linked on the website somewhere?

Reply:
The site links to their Itch.io profile, which is where the game is hosted.

Reply:
ok that's enough.